### PR TITLE
Updated for Django 1.7

### PR DESCRIPTION
**Pull request by micahhausler**
@Wilduck @wesleykendall 

- Added Django 1.7 compatibility 
- Bumped version to 0.3.2

### CODE REVIEW

File: entity_event/tests/context_loader_tests.py [hunk 1]
```diff
@@ -271,12 +271,12 @@ def test_dict_find_deep_nested_list_key(self):
...
 
 
-class GetModelIdsToFetchTest(SimpleTestCase):
+class GetModelIdsToFetchTest(TestCase):
     def test_no_events(self):
         self.assertEquals(context_loader.get_model_ids_to_fetch([], {}), {})
 
     def test_no_context_hints(self):
-        e = N(models.Event, context={}, persist_dependencies=False)
+        e = G(models.Event, context={})
```
Comment (wesleykendall) [inline entity_event/tests/context_loader_tests.py]: There a reason this had to be changed to G? And the other tests?

**Comment by wesleykendall** [comment]
@micahhausler had one comment

File: entity_event/tests/context_loader_tests.py (hunk 1 above)
Comment (micahhausler) [inline entity_event/tests/context_loader_tests.py]: Yes. The test was passing before due to a bug in Django 1.6 that was fixed in 1.7 and caused a `TypeError: Model instances without primary key value are unhashable` for 1.7. Changing to `G()` fixed the test.

http://stackoverflow.com/questions/26081146/django-1-7-passing-around-unsaved-instances-throws-unhashable-exception

Comment (wesleykendall) [inline entity_event/tests/context_loader_tests.py]: you'll just need to add id=1 to those N models. Can you do that so that the tests dont hit the db?

**Pull request merged**